telos-frontend/src/dnd/WidgetTypes.js

```diff
@@ -6,4 +6,5 @@ export const WidgetTypes = {
   CLOCK: 'clock',
   YOUTUBE_PLAYER: 'youtube_player',
   WEATHER: 'weather',
+  Steam: 'steam',
```

Comment:
Try to keep consistent with the other `WidgetTypes` and rename it to `STEAM`

Reply:
Nice spotting! I have resolved the issue.